release/pipline/dags/istio_daily_flow.py

```diff
@@ -0,0 +1,26 @@
+"""Airfow DAG used is the daily release pipline."""
+"""Copyright 2017 Istio Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from airflow import DAG
+
+import istio_common_dag
```

Comment:
Where does the istio tag get imported from?  I'm not clear if should also be included with this change or if something about airflow requires it to be stored elsewhere.